### ISSUE TITLE
todo.txt file on Dropbox...

### ISSUE THREAD

**Issue by trkkazulu** [sbs20/actiontext-win10#1]
Hi, 

I cannot add my todo.txt file from Dropbox. Is this a known issue or am i doing something wrong? 

Thanks, 

J. Wells

**Comment by sbs20** [comment]
Hi. I'm afraid it doesn't actually have any dropbox integration built in. I wrote it to work on a desktop with dropbox installed separately, so the app only ever reads and saves a local file (dropbox takes care of the rest). It's something I *might* add in time, but probably not very quickly I'm afraid.

What device are you using, a phone?

Sam

**Comment by trkkazulu** [comment]
Sam,
Thanks for getting back to me on this. I'm on both phone and a Windows 10 machine. I had a quick look at your source code. Maybe I'll add it myself. We'll see. 
Thank,
Jair-Rôhm 


Sent from my T-Mobile 4G LTE Device
-------- Original message --------From: Sam Strachan <notifications@github.com> Date: 5/7/17  17:52  (GMT+00:00) To: sbs20/actiontext-win10 <actiontext-win10@noreply.github.com> Cc: trkkazulu <trkkazulu2@yahoo.com>, Author <author@noreply.github.com> Subject: Re: [sbs20/actiontext-win10] todo.txt file on Dropbox... (#1) 
Hi. I'm afraid it doesn't actually have any dropbox integration built in. I wrote it to work on a desktop with dropbox installed separately, so the app only ever reads and saves a local file (dropbox takes care of the rest). It's something I might add in time, but probably not very quickly I'm afraid.
What device are you using, a phone?
Sam

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or mute the thread.


  
  




{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/sbs20/actiontext-win10","title":"sbs20/actiontext-win10","subtitle":"GitHub repository","main_image_url":"https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in GitHub","url":"https://github.com/sbs20/actiontext-win10"}},"updates":{"snippets":[{"icon":"PERSON","message":"@sbs20 in #1: Hi. I'm afraid it doesn't actually have any dropbox integration built in. I wrote it to work on a desktop with dropbox installed separately, so the app only ever reads and saves a local file (dropbox takes care of the rest). It's something I *might* add in time, but probably not very quickly I'm afraid.\n\nWhat device are you using, a phone?\n\nSam"}],"action":{"name":"View Issue","url":"https://github.com/sbs20/actiontext-win10/issues/1#issuecomment-299718938"}}}
